### PR TITLE
Unit tests for mount, mountv1 and doc commands

### PR DESCRIPTION
**Pull request by souravgupta-msft**

### CODE REVIEW

File: blobfuse2-code-coverage.yaml [hunk 1]
```diff
@@ -153,6 +153,32 @@ stages:
...
+                rm -rf $(MOUNT_DIR)/*
+                rm -rf $(TEMP_DIR)/*
+                cp $(BLOBFUSE2_CFG) /tmp/configBlockProfilerTemp.yaml
+                echo "dynamic-profile: true" >> /tmp/configBlockProfilerTemp.yaml
+                echo "cpu-profile: /tmp/cpuBlockProfTmp" >> /tmp/configBlockProfilerTemp.yaml
+                echo "mem-profile: /tmp/memBlockProfTmp" >> /tmp/configBlockProfilerTemp.yaml
+                echo "health_monitor:" >> /tmp/configBlockProfilerTemp.yaml
+                echo "  enable-monitoring: true" >> /tmp/configBlockProfilerTemp.yaml
+                echo "  monitor-disable-list:" >> /tmp/configBlockProfilerTemp.yaml
+                echo "    - bobfuse_stats" >> /tmp/configBlockProfilerTemp.yaml
```
Comment (gapra-msft) [inline blobfuse2-code-coverage.yaml:167]:
```suggestion
                echo "    - blobfuse_stats" >> /tmp/configBlockProfilerTemp.yaml
```

Comment (souravgupta-msft) [reply blobfuse2-code-coverage.yaml]: Done

**Review by vibhansa-msft — APPROVED** [review]

File: cmd/mount.go [hunk 2]
```diff
@@ -134,15 +133,16 @@ func (opt *mountOptions) validate(skipEmptyMount bool) error {
...
-	if opt.Debug {
-		_, err := os.Stat(opt.DebugPath)
-		if os.IsNotExist(err) {
-			err := os.MkdirAll(opt.DebugPath, os.FileMode(0755))
-			if err != nil {
-				return fmt.Errorf("invalid debug path [%s]", err.Error())
-			}
-		}
-	}
+	// commented as exectime is commented out
```
Comment (gapra-msft) [inline cmd/mount.go:136]: Should we just remove this code?

**Review by gapra-msft — APPROVED** [review]

File: cmd/mount.go (hunk 2 above)
Comment (souravgupta-msft) [reply cmd/mount.go]: Removed

**Pull request merged**